Sources/Support/DebugUI/DebugContentViews.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DebugViewContent.swift
+//
+//  Created by Nacho Soto on 5/30/23.
+
+#if DEBUG && os(iOS) && swift(>=5.8)
```

Comment:
Should we support macos as well? It probably shouldn't be a bottom sheet though... We can think about it and add in a future PR though